main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    create_data = {
+        "genres": ["Western", "Action", "Dramma"],
+        "actors": [
+            ("George", "Klooney"),
+            ("Kianu", "Reaves"),
+            ("Scarlett", "Keegan"),
+            ("Will", "Smith"),
+            ("Jaden", "Smith"),
+            ("Scarlett", "Johansson"),
+        ]
+    }
+
+    for genre in create_data.get("genres"):
+        Genre.objects.create(name=genre)
+
+    for first_name, last_mane in create_data.get("actors"):
+        Actor.objects.create(first_name=first_name, last_name=last_mane)
```

Comment:
There is a typo in the variable name `last_mane`. It should be `last_name` to match the intended use and the field name in the `Actor` model.